lib/logger.js

```diff
@@ -376,6 +373,10 @@ class Logger {
     }
 
     close() {
+        if (this._logger && this._errorHandler && this._logger.removeListener) {
```

Comment:
Hm. At first I was thinking this is too many conditions, but actually, it's not. The error handler will only be set up for the root logger and not the child loggers. but given that all the logger children share an internal Bunyan logger, calling a close method of a child will destroy the whole hierarchy. No need to take action now, it's a separate issue I've just noticed and we will deal with it separately.

Reply:
Ok